### DISCUSSION TITLE
sc-im floating point precision

### DISCUSSION THREAD

**Discussion by poetaman** [andmarti1424/sc-im#561] [Q&A]
What is the precision of floating point operations in sc-im? If it is lower than what other spreadsheet programs offer, can it be increased?

**Comment by andmarti1424** [comment]
It should be the precision of the C `double` type in the architecture where you run sc-im.

**Reply by poetaman** [reply]
@andmarti1424 Thanks, I also see you have a ``@round()`` function!
